packages/@expo/fingerprint/src/Fingerprint.types.ts

```diff
@@ -134,6 +134,11 @@ export interface Options {
    * Whether to include verbose debug info in source output. Useful for debugging.
    */
   debug?: boolean;
+
+  /**
+   * A custom hook function to transform file content sources before hashing.
+   */
+  fileHookTransform?: FileHookTransformFunction;
```

Comment:
Just one question: If I put a `console.log` inside of my transform, it doesn't print when using the CLI, correct? (due to https://github.com/expo/expo/blob/main/packages/%40expo/fingerprint/cli/src/utils/withConsoleDisabledAsync.ts).

Are there any other side effects / things that would corrupt the JSON shell output that we should be concerned about in this function? I guess there's not much beyond `withConsoleDisabledAsync` that we can do anyways.

Reply:
that's correct! `withConsoleDisabledAsync` still works even if people having console.log in the transform function.